resonate/scheduler.py

```diff
@@ -555,7 +575,14 @@ def _apply_resume(self, node: Node[State], promise: DurablePromise) -> None:
         assert promise.completed, "Promise must be completed."
 
         match node.value, promise:
-            case Enabled(Suspended(Rfnc(suspends=suspends) as f)) | Blocked(Running(Init(next=Rfnc(suspends=suspends) as f))), promise:
+            case Blocked(Running(Init(Rfnc() as next, suspends))), promise:
```

Comment:
Another cool find by DST. It is possible for a resume message to be received while we are trying to create the promise. I believe it is technically possible to receive a message from a message source _before_ a response from the create promise request comes back. In this case we need unblock all suspended nodes with an AWT rather than the promise value.